example/bufferCopy/src/bufferCopy.cpp

```diff
@@ -32,72 +32,55 @@ ALPAKA_FN_ACC size_t linIdxToPitchedIdx(size_t const globalIdx, size_t const pit
 //! Prints all elements of the buffer.
 struct PrintBufferKernel
 {
-    template<typename TAcc, typename TData, typename TExtent>
+    template<typename TAcc, typename TMemoryHandle, typename TData, typename TIdx>
```

Comment:
This is the `bufferCopy` example. Maybe showcase the `BufferAccessor` here? This would also save two template parameters since `TMemoryHandle` and `TIdx` are no longer necessary.

Reply:
Good idea!